AutoRest/Generators/CSharp/CSharp.Tests/Expected/Mirror.Primitives/Models/Product.cs

```diff
@@ -168,6 +176,11 @@ public Product(int? integer = default(int?), int? intProperty = default(int?), l
 
         /// <summary>
         /// </summary>
+        [JsonProperty(PropertyName = "base64urlArray")]
```

Comment:
I don't think Autorest currently adds the attribute when a property is an array of the type. In the same file, Date property has the [JsonConverter(typeof(DateJsonConverter))] attribute, but DateArray property does not.